data/hai/hai reveal 4 middle.txt

```diff
@@ -996,9 +1019,14 @@ mission "Hai Reveal [A08] Symbol Identification"
 			label newpictures
 			`	She waves a hand dismissively at her screen. "Of course, I only received the one image to refer to, and the older records are not necessarily very good quality or captured with future research librarians in mind. So, please, show me what else you have."`
+			choice
+				`	(Show her new pictures from your datapad.)`
 			`	You pull out your datapad and hand it to her with the folder of relevant pictures, some carefully redacted of Hai paraphernalia, for her to look at. As she begins flicking through them she makes some comments.`
-			`	"You know, there's quite a lot more of these than I expected. All quite recent too judging by the file quality and also by that crate design. They really are exactly the same as the original. Someone clearly has made a deliberate decision to reuse the exact symbol, no re-imagining or stylistic alterations whatsoever. If you ask me that sounds like a group that has set itself to having the same goals, which means the Syndicate would probably be very interested in this. Though I suppose you'd have the connections there if... Oh!" She pauses. "Now this is interesting."`
-			`	She turns the datapad to you and shows you what looks like an amulet or maybe a jewelry charm. The scale in the image isn't very clear. "Do you see this patterning around the outside of the symbol on this? I'm quite sure nearly everyone in the galaxy would simply pass that off as decorative, but it's not. This is an archaic written shorthand notation used for ship's logs, back when they were considering data storage requirements for the thousands of people on board the slow ships they sent to Alpha Centauri. You'd be amazed how much space just logs and diaries can take up when the journey takes as long as theirs did." She smiles at you, "As it happens I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!"`
+			`	"You know, there's quite a lot more of these than I expected. All quite recent too judging by the file quality and that crate design. They are exactly the same as the original. Someone clearly has made a deliberate decision to reuse the exact symbol, no re-imagining or stylistic alterations whatsoever. If you ask me, that sounds like a group that has set itself to having the same goals, which means the Syndicate would probably be very interested in this. Though I suppose you'd have the connections there if... Oh!" She pauses. "Now this is interesting."`
+			`	She turns the datapad to you and shows you what looks like an amulet or maybe a jewelry charm. The scale in the image isn't very clear. "Do you see this patterning around the outside of the symbol on this? I'm quite sure nearly everyone in the galaxy would simply pass that off as decorative, but it's not.`
+			choice
+				`	(Examine the symbols.)`
```

Comment:
This might be one break-up more than we really need here.

Reply:
I need to revisit "Examine the symbols" since Amazinite wants me to remove any "action choices."

Reply:
When I take it out, it becomes a text wall. However, I replaced "Examine the Symbols" with a more meaningful choice where you can impress Giti with your insight.

I just made major changes to the conversation to remove all the action choices. You should re-read it in its entirety.